toolkit/tools/imagegen/configuration/kernelcommandline.go

```diff
@@ -11,6 +11,10 @@ import (
 	"strings"
 )
 
+const (
+	SELinuxPolicyDefault = "selinux-policy"
```

Comment:
would this perhaps be a better fit in configuration/selinux.go?

Reply:
No `selinux.go` contains the SELinux mode (confusing called just `SELinux`), not SELinux stuff in general.

Reply:
okay, and why does this belong in kernelcommandline?

Reply:
Because `kernelcommandline.go` has the `KernelCommandLine.SELinuxPolicy` property, which this constant is directly related to.